src/components/CustomInputs/DictInput.vue

```diff
@@ -70,7 +70,10 @@ export default {
     value() {
       const dict = {}
       this.keys.forEach((k, i) => {
-        if (k && (!this.includeCheckbox || this.includedKeys.includes(k))) {
+        if (
+          (k && (!this.includeCheckbox || this.includedKeys.includes(k))) ||
+          Object.keys(JSON.parse(this.jsonInput)).includes(k)
+        ) {
           dict[k] = this.values[i]
         }
```

Comment:
```suggestion
        if (
          (k && (!this.includeCheckbox || this.includedKeys.includes(k))) ||
          (this.json && Object.keys(JSON.parse(this.jsonInput)).includes(k))
        ) {
          dict[k] = this.values[i]
        }
```

Reply:
ahhhh it was checking if they were on the json editor then push the keys/values but if not push the keys/values from the checkboxes